ddl/db_partition_test.go

```diff
@@ -1252,6 +1253,10 @@ func (s *testIntegrationSuite2) TestPartitionCancelAddPrimaryKey(c *C) {
 }
 
 func (s *testIntegrationSuite4) TestPartitionCancelAddIndex(c *C) {
+	if runtime.GOOS == "windows" {
+		// TODO: find the cause of the failure.
```

Comment:
It's better to create a github issue for it.

Reply:
No time to do this yet. I want to check these skipped tests after https://github.com/pingcap/tidb/projects/32.